coalib/parsing/filters/CanDetectFilter.py

```diff
@@ -0,0 +1,14 @@
+from coalib.parsing.filters.Filter import filter
+
+
+@filter
+def can_detect_filter(bear, args):
+    """
+    Filters the bears by ``LANGUAGES``.
+
+    :param bear:        Bear object.
+    :param args:        Set of detects on which bear is to be filtered.
+    :return:            ``True`` if this bear is to be displayed for given
+                        args, ``False`` otherwise.
```

Comment:
> ``True`` if this bear is to be displayed for given args, ``False`` otherwise.

Maybe rephrase this into something more general^^

> ``True`` if this bear matches the criteria inside ``args``, ``False`` otherwise.

Up to you :+1: